config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: kubeflow-system
+namespace: kfserving-system
```

Comment:
I have mentioned kubeflow-system here, https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving-from-default, can you help fix it ?